### PR TITLE
feat(Alert): added new component Alert

### PR DESCRIPTION
**Pull request by IsaevAlexandr**
Just new component for interface.
Start review from `docs` section in Storybook

related to DATAUI-1590

### CODE REVIEW

**Review by korvin89 — COMMENTED** [review]
Will u add some tests for basic functionality?

File: src/components/Alert/types.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    message?: React.ReactNode;
+    theme?: 'normal' | 'info' | 'positive' | 'warning' | 'danger';
+    /**
+     * Override default icons
+     */
+    icon?: React.ReactNode;
+    /**
+     * Zero border radius in card
+     */
+    squared?: boolean;
```
[the remark this answers is not in the document]
Comment (IsaevAlexandr) [reply src/components/Alert/types.ts:15]: you can change border radius by `className` any time. 

ок, i can change api with squared corners with `pin` api.

I think the best way to work with border radius is `theme` there you can set border radius for all components in one time

**Review by korvin89 — APPROVED** [review]

File: src/components/Alert/AlertIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    CircleInfo,
+    CircleInfoFill,
+    CircleXmark,
+    CircleXmarkFill,
+    TriangleExclamation,
+    TriangleExclamationFill,
+} from '@gravity-ui/icons';
+
+import {Icon} from '../Icon';
+import type {SVGIconData} from '../Icon/types';
```
Comment (amje) [inline src/components/Alert/AlertIcon.tsx:15]: There is type `IconData` in Icon file

File: src/components/Alert/constants.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import {block} from '../utils/cn';
+
+import './Alert.scss';
+
+export const DEFAULT_ICONS_SIZE = 18;
```
Comment (amje) [inline src/components/Alert/constants.ts:5]: `_ICON_`, in singular

File: src/components/Alert/types.ts [hunk 4]
```diff
@@ -0,0 +1,86 @@
+export type AlertTypes = 'normal' | 'info' | 'positive' | 'warning' | 'danger';
```
Comment (amje) [inline src/components/Alert/types.ts:1]: `AlertTheme`?

File: src/components/Alert/types.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+export type AlertView = 'filled' | 'outlined';
+
+export interface AlertProps {
+    title?: React.ReactNode;
+    message?: React.ReactNode;
+    theme?: 'normal' | 'info' | 'positive' | 'warning' | 'danger';
```
Comment (amje) [inline src/components/Alert/types.ts:7]: use alias from above

File: src/components/Alert/types.ts [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    /**
+     * Override default icons
+     */
+    icon?: React.ReactNode;
+    /**
+     * @default - normal
+     */
+    corners?: 'normal' | 'none';
```
Comment (amje) [inline src/components/Alert/types.ts:15]: What about different naming? `cornerStyle` with values: `rounded` | `square`

File: src/components/Alert/types.ts [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+     * )
+     * ```
+     */
+    actions?: React.ReactNode | AlertAction[];
+    /**
+     * Center all content in vertical direction,
+     * useful if for some reason you actions takes more space then text
+     * or needed icon to be on the middle of the card
+     */
+    contentCenter?: true;
```
Comment (amje) [inline src/components/Alert/types.ts:60]: Maybe `centered: boolean`?

**Review by amje — CHANGES_REQUESTED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**